templates/word-daily-level/src/addons/words/keyboard.js

```diff
@@ -34,33 +76,37 @@ export class Keyboard extends EventTarget {
                 this.dispatchEvent(new CustomEvent("key", { detail: key }));
             }
         });
+        
+        this.innerHTML = template;
     }
+
     setState({words, marks}) {
-        const charStates = Array.from(Array(26), () => Grid.EMPTY);
+        const charStates = Array.from(Array(26), () => EMPTY);
         const ccA = "A".charCodeAt(0);
 
         for (let w = 0; w < marks.length; w++) {
             for (let i = 0; i < words[w].length; i++) {
                 const c = words[w].charCodeAt(i) - ccA;
-                const newMarkPriority = Grid.GuessSymbols.indexOf(marks[w][i]);
+                const newMarkPriority = GuessSymbols.indexOf(marks[w][i]);
 
-                if (newMarkPriority > Grid.GuessSymbols.indexOf(charStates[c])) {
+                if (newMarkPriority > GuessSymbols.indexOf(charStates[c])) {
                     charStates[c] = marks[w][i];
                 }
             }
         }
 
-        const keyboard = document.getElementsByClassName("keyboard")[0];
-        for (let row = 0; row < keyboard.children.length; row++) {
-            for (let btn = 0; btn < keyboard.children.item(row).children.length; btn++) {
-                const key = keyboard.children.item(row).children.item(btn);
+        for (let row = 0; row < this.children.length; row++) {
+            for (let btn = 0; btn < this.children.item(row).children.length; btn++) {
+                const key = this.children.item(row).children.item(btn);
                 if (key.textContent.length === 1) {
                     const ccId = key.textContent.charCodeAt(0) - ccA;
-                    key.classList = [Grid.GuessClasses[Grid.GuessSymbols.indexOf(charStates[ccId])]];
+                    key.setAttribute("s", charStates[ccId]);
                 }
             }
         }
-
-
     }
 }
+
+export const KeyboardTag = "word-game-keyboard";
```

Comment:
UpperCamelCase should be only for classes/types. Can we use lowerCamelCase here?

```suggestion
export const keyboardTagName = "word-game-keyboard";
```